### PR TITLE
Add MakoLab issuers implementation

### PR DESCRIPTION
**Pull request by sdml**
At the moment, there is a configuration issue on our end in the controller document, so I am creating this pull request just to ensure that the implementation specification is correct.

As soon as the controller document and verification method resolvers are working correctly at our endpoint, I will change this draft to a regular pull request.

### CODE REVIEW

**Comment by PatStLouis** [comment]
@sdml you might want to add the vc2.0 tag if you want to also run the vcdm test suite

**Comment by sdml** [comment]
Issuer endpoints are now published. I change this PR as Ready for review.

**Review by PatStLouis — APPROVED** [review]

**Review by PatStLouis — CHANGES_REQUESTED** [review]
`id` field is required on the issuer entries, and we recommend using a `did:key:` value. This will be injected as the issuer of the vc.

File: implementations/MakoLab.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "MakoLab",
+  "implementation": "MakoLab Verifiable Credentials",
+  "issuers": [
+    {
+      "id": "did:key:zDnaeipTBN8tgRmkjZWaQSBFj4Ub3ywWP6vAsgGET922nkvZz",
+      "endpoint": "https://db.makolab.pro/issuers/credentials/issue",
+      "method": "POST",
```
Comment (BigBlueHat) [inline implementations/MakoLab.json:8]:
```suggestion
      "method": "POST",
```

File: implementations/MakoLab.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      "supportedEcdsaKeyTypes": ["P-256"],
+      "supports": {
+        "vc": ["1.1", "2.0"]
+      },
+      "tags": ["ecdsa-rdfc-2019", "vc2.0"]
+    },
+    {
+      "id": "did:key:zDnaeipTBN8tgRmkjZWaQSBFj4Ub3ywWP6vAsgGET922nkvZz",
+      "endpoint": "https://db.makolab.pro/issuers/sd/credentials/issue",
+      "method": "POST",
```
Comment (BigBlueHat) [inline implementations/MakoLab.json:18]:
```suggestion
      "method": "POST",
```

**Review by BigBlueHat — CHANGES_REQUESTED** [review]
We don't need `method` anymore, but otherwise this looks good.

**Review by PatStLouis — APPROVED** [review]

File: implementations/MakoLab.json (hunk 1 above)
Comment (TallTed) [reply implementations/MakoLab.json:8]: @BigBlueHat — If `method` is no longer needed, shouldn't your suggestion be --
```suggestion
```

File: implementations/MakoLab.json (hunk 2 above)
Comment (TallTed) [reply implementations/MakoLab.json:18]: As above...
```suggestion
```

File: implementations/MakoLab.json (hunk 1 above)
Comment (BigBlueHat) [reply implementations/MakoLab.json:8]: Heh...yeah... Reviewing too fast. 🤦‍♂️

**Pull request merged**